composer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "spryker/checksum-generator",
+  "type": "library",
+  "description": "ChecksumGenerator module",
```

Comment:
If this is not supposed to be a normal core module, we must remove that word here.